corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -10,9 +10,10 @@
 When forms work with multiple datums, they need to be named in a way that is predictable for app builders, who
 reference them inside forms. This is most relevant to the "select parent first" feature and to parent/child
 modules.  See ``update_refs`` and ``rename_other_id``, both inner functions in ``add_parent_datums``, plus
-`this comment <https://github.com/dimagi/commcare-hq/blob/c9fa01d1ccbb73d8f07fefbe56a0bbe1dbe231f8/corehq/apps/app_manager/suite_xml/sections/entries.py#L966-L971>`_
-on matching parent and child datums.
+`this comment`_ on matching parent and child datums.
 
+
+.. _this comment: https://github.com/dimagi/commcare-hq/blob/c9fa01d1ccbb73d8f07fefbe56a0bbe1dbe231f8/corehq/apps/app_manager/suite_xml/sections/entries.py#L966-L971  # noqa
```

Comment:
A little hacky, I admit, but I have checked this and it works: When rendered, RST appends "#noqa" to the URL with no spaces. GitHub ignores the second "#", and the correct lines are highlighted.